__tests__/app.test.js

```diff
@@ -141,6 +141,26 @@ describe('GET /api', () => {
           });
       }); 
 
+      test('GET: 200 - returns articles filtered by topic', () => {
+        return request(app)
+          .get('/api/articles?topic=cats')
+          .expect(200)
+          .then(({ body }) => {
+            body.articles.forEach((article) => {
+              expect(article.topic).toBe('cats')
+            });
+          });
+      });
+
+      test('GET: 200 -returns an empty array when the topic does not exist', () => {
+        return request(app)
+          .get('/api/articles?topic=nothing')
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.articles).toEqual([]);
+          });
+      });
+
```

Comment:
this is a 404 topic not found error, should return an error message 
We also need a 200 when topic exist but no articles related - e.g. "paper", that should return an empty array